### ISSUE TITLE
How to upload the photo into blog?

### ISSUE THREAD

**Issue by skanel** [bianjp/blog-spring#3]
Could you help, how can I upload the photo of the blog post?

**Comment by bianjp** [comment]
This is not supported yet. :)

Though I'm really using it in my own site, it lacks many features and might not be suitable for others.

**Comment by skanel** [comment]
yes, can you share the feature of photo upload?  that is very important.

**Comment by bianjp** [comment]
Personally I uploaded pictures to my server manually by `scp`.

**Comment by skanel** [comment]
then how you show it in the blog post?

**Comment by bianjp** [comment]
The uploaded pictures will be served by Nginx, and I reference its link in the post.

**Comment by skanel** [comment]
but when I embeded the img tag is does render

for example I add: 

<img src="/uploads/2017/12/popup-dict-screenshot.png" alt="popup-dict screenshot">

in the body of post.

**Comment by skanel** [comment]
when I add this tag into body of post.

`<img src="https://en.wikipedia.org/wiki/Cullinan_Diamond#/media/File:FGS_Wells_with_Cullinan_Diamond.jpg" alt="popup-dict screenshot">`

it not render as image?

**Comment by bianjp** [comment]
The post content will be parsed by AsciiDoctor. Reference [AsciiDoctor documentation](https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#images) for how to render images.
